### PR TITLE
Change report order & fix pagination bug

### PR DESCRIPTION
**Pull request by queq1890**
## Story URL
https://trello.com/c/qe5W8wK9/37-2-sprint3-reverse-report-list-order

### CODE REVIEW

File: src/containers/DailyReportPage/DailyReportContainer.js [hunk 1]
```diff
@@ -148,13 +164,18 @@ class DailyReportContainer extends Component {
...
+                        report={report}
+                        location={location}
+                        count={count}
+                        updateUserDailyReportsCount={updateUserDailyReportsCount}
+                      />
                     </ReportsRowColumn>
                   </ReportsRow>
                 ))}
               </ReportsTable>
               <button onClick={() => this.prevPage()}>◀︎Prev</button>{' '}
```
Comment (ggianglv) [inline src/containers/DailyReportPage/DailyReportContainer.js]: just this `<button onClick={this.prevPage}>◀︎Prev</button>{' '}`

**Comment by queq1890** [comment]
TODO : 
- fix variables
- research the error which appears when we delete a lot of reports from paginated page

File: src/containers/DailyReportPage/DailyReportContainer.js (hunk 1 above)
Comment (queq1890) [reply src/containers/DailyReportPage/DailyReportContainer.js]: Thanks, but I decided to give an argument to `prevPage()` function.
So, final code will look like 
`<button onClick={() => this.prevPage(currentPage)}>◀︎Prev</button>{' '}`

Comment (ggianglv) [reply src/containers/DailyReportPage/DailyReportContainer.js]: ok 👍

**Review by ggianglv — APPROVED** [review]
LGTM

**Review by phuongthuan — APPROVED** [review]
Greate stuff.

**Comment by queq1890** [comment]
@phuongthuan @gianglevan94 
Thank you so much !
I want to send Unipos to you guys but all Internship worker lost  accounts😂

Anyway, I merge this pull reqiest.

**Comment by phuongthuan** [comment]
> @phuongthuan @gianglevan94
> Thank you so much!
> I want to send Unipos to you guys but all Internship worker lost accounts😂
> 
> Anyway, I merge this pull request.

Unipos won't with Internship account. (lol)

**Pull request merged**